tests/annotation_merge.nf.test.snap

```diff
@@ -101,13 +101,16 @@
                 "test_VEP.ann.vcf.gz:md5,d41d8cd98f00b204e9800998ecf8427e",
                 "test_snpEff.ann.vcf.gz:md5,fd266e623365c7d1837bc9eb088fe70c",
                 "test_snpEff_VEP.ann.vcf.gz:md5,d41d8cd98f00b204e9800998ecf8427e"
+            ],
+            [
+                "WARN: Access to undefined parameter `consensus_vcfs` -- Initialise it to a default value eg. `params.consensus_vcfs = some_value`"
```

Comment:
@FriederikeHanssen we forgot something with this params

Reply:
Solving this comment for now as I created an issue for it https://github.com/nf-core/sarek/issues/2082